src/tools/runk/src/commands/exec.rs

```diff
@@ -0,0 +1,220 @@
+// Copyright 2021-2022 Kata Contributors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+use anyhow::anyhow;
+use anyhow::Result;
+use libcontainer::status::Status;
+use libcontainer::{
+    builder::ContainerBuilder, container::ContainerAction, status::get_current_container_state,
+};
+use liboci_cli::Exec;
+use oci::{ContainerState, Process as OCIProcess, Spec};
+use rustjail::container::update_namespaces;
+use slog::{debug, info, Logger};
+use std::fs::File;
+use std::path::{Path, PathBuf};
+
+pub async fn run(opts: Exec, root: &Path, logger: &Logger) -> Result<()> {
+    let status = Status::load(root, &opts.container_id)?;
+    let state = get_current_container_state(&status)?;
+    if let ContainerState::Paused | ContainerState::Stopped = state {
```

Comment:
Can here use a white list instead of a black list to check which state can run exec? For example, only goon when `state == running`.

If a container with a `Created` status will pass this check.